code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -13,7 +13,7 @@
 	idle_power_usage = 40
 	var/energy = 0
 	var/max_energy = 50
-	var/rechargerate = 2
+	var/rechargerate = 5

```

Comment:
whoops, that needs reverted, was a thingie I had for testing mixing locally without constantly running back and forth between dispensers waiting on recharges